app/waf/src/controllers/adc.controller.ts

```diff
@@ -479,8 +501,9 @@ export class AdcController extends BaseController {
     // install as3
     if (await this.adcStCtr.readyTo(AdcState.INSTALLED))
       await this.installAS3(adc);
-
     // create tenant
+    if (await this.adcStCtr.readyTo(AdcState.PARTITIONED))
+      await this.installPartition(adc);
```

Comment:
might want to consider the HW path around line#122.  we might want to make a decision about how to deal with HW now because it does not seem to install the partition for HW. TO me, HW path can be deleted.

Reply:
This is a legacy issue related to the HW design. Let's try to change this after the final decision is  done.

Reply:
ok. Another discussion and a future PR seems fine to me .